hazelcast-jet-core/src/test/java/com/hazelcast/jet/rocksdb/RocksMapTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.rocksdb;
+
+import com.hazelcast.internal.serialization.InternalSerializationService;
+import com.hazelcast.jet.JetException;
+import com.hazelcast.jet.core.JetTestSupport;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import static java.util.Collections.emptyMap;
+
+public class RocksMapTest extends JetTestSupport {
+    private RocksDBStateBackend rocksDBStateBackend;
+    private InternalSerializationService serializationService;
+    private RocksMap<String, Integer> rocksMap;
+
+
+    @Before
+    public void init() {
+        serializationService = getJetService(createJetMember())
+                .createSerializationService(emptyMap());
+        rocksDBStateBackend = new RocksDBStateBackend(serializationService);
+        rocksMap = rocksDBStateBackend.getMap();
+    }
+
+    @After
+    public void cleanup() {
+        rocksDBStateBackend.deleteKeyValueStore();
+        serializationService.dispose();
+    }
+
+
+    @Test
+    public void test_getMap() {
+        assert rocksMap != null;
```

Comment:
Unit tests don't use the Java `assert` feature. Use `Assert.assertNotNull()` in this case and other methods as appropriate, and write the messages which are important when a test fails. Also, this isn't a great test because if there's an exception in `getMap`, it will fail in the init phase and not in this test.